CRM/settings.py

```diff
@@ -118,7 +119,16 @@
 
 STATIC_URL = 'static/'
 STATICFILES_DIRS = [BASE_DIR / 'static']
+
+# Set the BASE_DIR to the root directory of your Django project
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+# Media settings
+MEDIA_URL = 'media/'
```

Comment:
also add media dir in gitignore as we should never commit such sort of directories